src/main/java/com/habicus/core/security/JWTAuthorizationFilter.java

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Can we add Java Doc to all the public functions in this class?
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html